traits/trait_converters.py

```diff
@@ -42,6 +42,33 @@ def trait_cast(obj):
     return None
 
 
+def as_ctrait(obj):
+    """ Convert to CTrait if the object knows how, else raise TraitError.
+
+    Parameters
+    ----------
+    obj: An object that supports conversion to `CTrait`. Refer
+        documentation for when the object supports this conversion.
+
+    Returns
+    -------
+    ctrait.CTrait
+
+    Raises
+    ------
+    TypeError if the object does not support conversion to CTrait.
```

Comment:
Also needs an update for numpydoc style: the "if ..." part should be on the following line, and indented.

Once you've updated, it would be worth double checking that the result of the Sphinx build looks right.